include/ignition/common/PluginPtr.hh

```diff
@@ -218,11 +220,11 @@ namespace ignition
 
     /// \brief Typical usage for TemplatePluginPtr is to just hold a generic
     /// Plugin type.
-    using PluginPtr = TemplatePluginPtr<Plugin>;
+    using PluginPtr IGN_DEPRECATED(5) = TemplatePluginPtr<Plugin>;
```

Comment:
this may have broken the windows build:

[![Build Status](https://build.osrfoundation.org/buildStatus/icon?job=ign_common-pr-win&build=396)](https://build.osrfoundation.org/job/ign_common-pr-win/396/) https://build.osrfoundation.org/job/ign_common-pr-win/396/

Reply:
reverted